### DISCUSSION TITLE
Unclear - where do we put the username?

### DISCUSSION THREAD

**Discussion by anewphilip** [rdeepak2002/reddit-place-script-2022#220] [Q&A] [answered]
Sorry in advance if I'm being a tard. I'm a bit unfamiliar with Python and I honestly just want to help my friends put something on r/place. Reading the steps for this, I'm a bit puzzled as where to put the username for a worker. 

`{
  "image_path":"image.png",
  "image_start_coords": [741, 610],
  "thread_delay": 2,

  "workers": {
    "worker1username": {
      "password": "password",
      "start_coords": [0, 0]
    },
    "worker2username": {
      "password": "password",
      "start_coords": [0, 50]
    }
  }
}`

With password, it seems you put the password in quotes after the colon. e.g "password": "thisismypassword". But with the username, there's no field to enter the username into following the colon. Do we put the username in the same way we put in the password? E.G "worker2username": "thisismyaccountname"?

**Comment by JAKONL** [comment, marked as answer]
You got it! replace `worker1username` and `worker2username` with the usernames of the accounts you want to utilize.

If you want more than two accounts then copy the following and paste it after the `}` that is the 3rd to last.
```
,
"worker2username": {
"password": "password",
"start_coords": [0, 50]
}

```

If you have problems try using a JSON Linter online like this one: https://jsonlint.com

**Reply by anewphilip** [reply]
Thank you!
